options/options.c

```diff
@@ -52,6 +52,79 @@
 #include "stream/stream.h"
 #include "demux/demux.h"
 
+// Windows specific defines:
+#ifndef DWMWCP_DEFAULT
```

Comment:
Those ifndefs are pointless, those are enums in upstream, so it won't work anyway.